oca_dependencies.txt

```diff
@@ -1 +1,2 @@
-server-ux
+server-tools
+server-ux
```

Comment:
for v12, should be only server-tools, as base_exception resides there.
Isnt it  or we miss anything?

Reply:
@JayVora-SerpentCS I guess server-ux is usefull for another module of this repository.

Reply:
@JayVora-SerpentCS ux is already in 12.0 branch 
https://github.com/OCA/purchase-workflow/blob/12.0/oca_dependencies.txt